packages/server/docs/openapi.yaml

```diff
@@ -0,0 +1,1933 @@
+        schedule:
+          $ref: '#/components/schemas/MetaSchedule'
+        services:
+          $ref: '#/components/schemas/MetaServicesArr'
+        streetViewImage:
+          $ref: '#/components/schemas/MetaStreetViewImage'
+        subcategories:
+          $ref: '#/components/schemas/SubCategoryWithResources'
+        website:
+          $ref: '#/components/schemas/MetaWebsite'
+    ResourceDraft:
+      type: object
+      properties:
+        _id:
+          $ref: '#/components/schemas/Meta_id'
+        address:
+          $ref: '#/components/schemas/MetaAddress'
+        createdAt:
+          $ref: '#/components/schemas/MetaDateTime'
+        createdBy:
+          $ref: '#/components/schemas/MetaCreatedBy'
+        deleted:
+          $ref: '#/components/schemas/MetaDeleted'
+        description:
+          $ref: '#/components/schemas/MetaDescription'
+        kudos:
+          type: integer
+          default: 0
+        latitude:
+          $ref: '#/components/schemas/MetaLatitude'
+        legacyId:
+          $ref: '#/components/schemas/MetaLegacyId'
+        longitude:
+          $ref: '#/components/schemas/MetaLongitude'
+        name:
+          type: string
+          default: 'Community Food Share'
+        phone:
+          $ref: '#/components/schemas/MetaPhone'
+        schedule:
+          $ref: '#/components/schemas/MetaSchedule'
+        services:
+          $ref: '#/components/schemas/MetaServicesArr'
+        streetViewImage:
+          $ref: '#/components/schemas/MetaStreetViewImage'
+        subcategories:
+          type: array
+          items:
+            type: object
+            properties:
+              _id:
+                $ref: '#/components/schemas/Meta_id'
+              name:
+                type: string
+                example: 'Food Pantries'
+              stub:
+                type: string
+                example: 'food_pantries'
+              parentCategory:
+                allOf:
+                  - $ref: '#/components/schemas/CategoryData'
+                  - type: object
+                    properties:
+                      subcategories:
+                        type: array
+                        items:
+                          $ref: '#/components/schemas/Meta_id'
+              createdAt:
+                $ref: '#/components/schemas/MetaDateTime'
+              lastModifiedAt:
+                $ref: '#/components/schemas/MetaDateTime'
+              __v:
+                type: integer
+        website:
+          $ref: '#/components/schemas/MetaWebsite'
+    SlackBotResponse:
+      type: object
+      properties:
+        blocks:
+          type: array
+          items:
+            type: object
+            properties:
+              text:
+                type: object
+                properties:
+                  text:
+                    type: string
+                    example: 'I found you in UpSwyng, but you have to be an administrator to tell me to run this command'
```

Comment:
Thoughts on providing the "true" [successful return](https://github.com/CodeForBoulder/upswyng/blob/master/packages/server/src/routes/api/bot/check-links.ts#L9)? It feels odd that we're returning 200s for unknown or  unauthorized users though.

Reply:
Ah, I see this is used as the generic bot response though 🤔 Maybe a generic "starting job" message instead?